### DISCUSSION TITLE
Sqlite Usage

### DISCUSSION THREAD

**Discussion by abhinavsharma1212** [invertase/react-native-firebase#7392] [General]
Does firebase crashlytics uses the sqlite database? Because we got the VAPT issue (Use of Vulnerable SQLite Database and execution of Raw SQL Queries), after the implementation of crashlytics. If yes please let us know how to resolve this and provide the proper evidence for it.

**Comment by pavanbora1998** [comment]
I am also facing the same issue While doing VAPT testing.

**Comment by nancyjainpanamax** [comment]
Hello All Even am facing the same issue of VAPT because of use of Firebase crashlytics as such they are giving error of raw queries used in VAPT so please anyone have any evidence then please share here
